ci-operator/step-registry/hypershift/mce/aws/destroy/2.2/hypershift-mce-aws-destroy-2.2-chain.yaml

```diff
@@ -0,0 +1,44 @@
+chain:
+  as: hypershift-mce-aws-destroy-2.2
+  steps:
+    - as: destroy
+      cli: latest
+      env:
+        - name: HYPERSHIFT_BASE_DOMAIN
+          default: "origin-ci-int-aws.dev.rhcloud.com"
+          documentation: "The cluster's FQDN will be a subdomain of the base domain."
+        - name: HYPERSHIFT_AWS_REGION
+          default: "us-east-1"
+          documentation: "The AWS region of the cluster."
+      commands: |-
+        set -exuo pipefail
+
+        CLUSTER_NAME="$(echo -n $PROW_JOB_ID|sha256sum|cut -c-20)"
+        
+        bin/hypershift dump cluster --artifact-dir=$ARTIFACT_DIR \
+          --dump-guest-cluster=true \
+          --namespace local-cluster \
+          --name="${CLUSTER_NAME}"
+      
+        echo "$(date) Deleting HyperShift cluster ${CLUSTER_NAME}"
+        bin/hypershift destroy cluster aws \
+          --aws-creds=${CLUSTER_PROFILE_DIR}/.awscred  \
+          --name ${CLUSTER_NAME} \
```

Comment:
do we not need the infra-id argument here as well?

Reply:
no need to set infra-id when destroying a hosted cluster  [Delete a HostedCluster](https://hypershift-docs.netlify.app/getting-started/#:~:text=%3D%24NODEPOOL_REPLICAS-,Delete%20a%20HostedCluster,-To%20delete%20a)